bundles/org.openhab.binding.bondhome/src/main/java/org/openhab/binding/bondhome/internal/handler/BondDeviceHandler.java

```diff
@@ -205,6 +205,27 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 }
                 break;
 
+            case CHANNEL_RAW_FAN_SPEED:
+                logger.trace("Fan raw speed command");
```

Comment:
please remove this log line; it's easy to get the same information from the events log (yes, I realize the other channels do it; they should also probably be removed)